dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/passwordreset/PasswordResetData.java

```diff
@@ -81,9 +84,7 @@ public PasswordResetData(BuiltinUser aBuiltinUser) {
         token = UUID.randomUUID().toString();
         long nowInMilliseconds = new Date().getTime();
         created = new Timestamp(nowInMilliseconds);
-        long ONE_MINUTE_IN_MILLISECONDS = 60000;
-        long futureInMilliseconds = nowInMilliseconds + (SystemConfig.getMinutesUntilPasswordResetTokenExpires() * ONE_MINUTE_IN_MILLISECONDS);
-        expires = new Timestamp(new Date(futureInMilliseconds).getTime());
+        expires = created;
```

Comment:
I think we should remove it, that way we will enforce setting expire field. Since if it will not be set there will be an exception thrown that the field cannot be null when trying to set it to DB. Otherwise we may end up with token that is essentially useless during creation.